src/Components/DynamicFields.res

```diff
@@ -388,7 +394,20 @@ let make = (
         | InfoElement
         | None => ""
         }
-        acc->Js.Dict.set(item.required_field, value->Js.Json.string)
+        if (
+          isSavedCardFlow &&
+          item.display_name === "card_holder_name" &&
+          (item.field_type === BillingName || item.field_type === FullName)
+        ) {
+          if !isAllStoredCardsHaveName {
+            acc->Js.Dict.set(
+              "payment_method_data.card_token.card_holder_name",
+              value->Js.Json.string,
```

Comment:
Where is this value coming from? Also are complete/empty handled for this case?

Reply:
<img width="541" alt="Screenshot 2023-11-29 at 4 36 45 PM" src="https://github.com/juspay/hyperswitch-web/assets/121166031/57c02177-d62f-4dd8-87c7-194389bfb3b5">

The value is coming from the screenshot.

Complete and empty is handled based on the fieldsArr which are input fields to be rendered, only while creating the requiredFieldsBody are we directly using the required fields object from the payment method list api